src/NexusMods.App.UI/Controls/Diagnostics/IDiagnosticEntryViewModel.cs

```diff
@@ -0,0 +1,15 @@
+using System.Reactive;
+using NexusMods.Abstractions.Diagnostics;
+using ReactiveUI;
+
+namespace NexusMods.App.UI.Controls.Diagnostics;
+
+public interface IDiagnosticEntryViewModel : IViewModelInterface
+{
+    string Summary { get; }
+    
+    DiagnosticSeverity Severity { get; }
+    
+    ReactiveCommand<Unit, Unit> SeeDetailsCommand { get; }
```

Comment:
This should be `ReactiveCommand<Unit, Diagnostic>` so it's easier to use with `MergeMany`.